proxy/http/HttpSM.cc

```diff
@@ -4186,6 +4186,8 @@ HttpSM::parse_range_and_compare(MIMEField *field, int64_t content_length)
   const char *s, *e, *tmp;
   RangeRecord *ranges = nullptr;
   int64_t start, end;
+  int64_t cutoff = 9223372036854775807 / 10;
+  int64_t cutlim = 9223372036854775807 % 10;
 
```

Comment:
Specifying the constant as 0x7FFFFFFFFFFFFFFF might be more meaningful.   Or better yet, perhaps a named constant.

Reply:
However, the divide and modulo 10 seem kind of arbitrary.  Are we just trying to work around a compiler bug?

Reply:
Or even a INT64_MAX might be nicer here ?

Reply:
INT64_MAX is a better choice.